src/Payouts/Generator/CLI.php

```diff
@@ -303,6 +318,71 @@ public function reset_payouts( array $args = null, array $assoc_args = null ) {
 
 	}
 
+	/**
+	 * Remove payouts, for a specified order
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args An array of arguments, most importantly the order (post) ID.
+	 * @param array $assoc_args (currently unused) additional passed arguments
+	 */
+	public function delete_payouts( array $args = null, array $assoc_args = null ) {
+		$order_id    = $args[0];
+
+		if ( empty( $order_id ) ) {
+			WP_CLI::error( sprintf( __( 'Order with ID %d does not exist.', 'tribe-cli' ), $order_id ) );
+		}
+
+		$this->delete_payout( $order_id );
+	}
+
+	/**
+	 * Update payouts, for a specified order.
+	 * Currently only updates status.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args An array of arguments, most importantly the order (post) ID.
+	 * @param array $assoc_args additional passed arguments
+	 */
+	public function update_payouts( array $args = null, array $assoc_args = null ) {
+		$order_id    = $args[0];
+		$status = $assoc_args['status'];
+
+		if ( empty( $order_id ) ) {
+			WP_CLI::error( sprintf( __( 'Order with ID %d does not exist.', 'tribe-cli' ), $order_id ) );
+		}
+
+		if ( empty( $status ) ) {
+			WP_CLI::error( sprintf( __( 'A status to update to is required.', 'tribe-cli' ) ) );
+		}
+
+		$stati_map = [
+			'pending' => Payouts::STATUS_PENDING,
+	 		'pending-order' => Payouts::STATUS_PENDING_ORDER,
+	 		'paid' => Payouts::STATUS_PAID,
+	 		'failed' => Payouts::STATUS_FAILED,
+		];
+
+		$repository = tribe_payouts();
+		$repository->by( 'order', $order_id );
+		$repository->set_found_rows( true );
+		$found = $repository->found();
+
+		if ( 0 >= $found ) {
+			error_log('none found');
+			return;
+		}
+
+		$ids = $repository->get_ids();
+
+		foreach( $ids as $id ) {
+			$repository->set( 'post_status', $stati_map[ $status ] );
```

Comment:
This doesn't need to be within a `foreach`, the `set` will automatically set it for all posts in the repository.

Reply:
```suggestion
		$repository->set( 'post_status', $stati_map[ $status ] );
```